fili-core/src/main/java/com/yahoo/bard/webservice/web/endpoints/DataServlet.java

```diff
@@ -365,6 +365,7 @@ public void getData(
                     uriInfo,
                     this
             );
+            RequestLog.stopTiming("DataApiRequest");
```

Comment:
I'm telling you, when someone says "X is convenient" you should hear "X will cause unexpected bugs a few refactors from now."